File: vues/services.php
<?php
if ($host == 'localhost')
{

if(!empty($_GET['service']))
{
	if($_GET['value'] == 1)
	{
		system('sudo chkconfig '.$_GET['service'].' on');
	}
	if($_GET['value'] == 0)
	{
		system('sudo chkconfig '.$_GET['service'].' off');
	}
}



?>
<form>
<center><h2>Services on <?php echo $_GET['host']; ?></h2></center>
<?php

        echo"<center><table cellspacing='0' cellpadding='0'>";
        system('sudo perlScripts/services.pl');
        echo"</table></center>";
        echo"</div>";



}
?>
</form>

File: perlScripts/services.pl
#!/usr/bin/perl

open(PROCESS, "chkconfig -l |");

my $i = 0;

while (defined(my $ligne = <PROCESS>))
{	
	chomp($ligne);
	@items = split(/ +/, $ligne);
	print "<tr><td style='width:90%;'><strong>$items[0]</strong></td><td>";
	if ($ligne =~ m/on/)
	{
		print'<a rel="external" href="index.php?page=services&host=localhost&service=';
		print $items[0];
		print'&value=0" data-theme="b" data-role="button" data-inline="true">ON</a>';	
	}
	else 
	{
		print'<a rel="external" href="index.php?page=services&host=localhost&service=';
		print $items[0];
		print'&value=1" data-theme="d" data-role="button" data-inline="true">OFF</a>'; 
	}
	print "</td></tr>\n";
	$i++;
}
